app/components/add-task.hbs

```diff
@@ -0,0 +1,4 @@
+<Input @value={{this.message}} class='add-input'/>
+<button {{on "click" (fn this.addMessage this.message)}} class='add-input-button'>{{yield}}</button>
```

Comment:
wrap the component in a form. form should have submit button, on submit it must save model.

Reply:
message should not be used